terraform/variables.tf

```diff
@@ -141,6 +141,12 @@ variable "acm_secret_type" {
   type        = string
 }
 
+variable "acm_source_repository_fqdns" {
```

Comment:
Can you add this variable in the README step that talks about authenticating ACM?